assets/tests/components/map.test.tsx

```diff
@@ -194,6 +203,75 @@ describe("map", () => {
     expect(container.innerHTML).not.toContain("m-garage-icon")
   })
 
+  test("doesn't draw station icons at zoom levels < 15", async () => {
+    const mapRef: MutableRefObject<LeafletMap | null> = { current: null }
+
+    const { container } = render(
+      <Map vehicles={[vehicle]} reactLeafletRef={mapRef} stations={[station]} />
+    )
+
+    // Manual zoom
+    act(() => {
+      mapRef.current!.setZoom(14)
+    })
+    await animationFramePromise()
+    expect(container.innerHTML).not.toContain("m-station-icon")
+    expect(screen.queryByText(station.name)).toBeNull()
+  })
+
+  test("draws station icons at zoom levels >= 15", async () => {
+    const mapRef: MutableRefObject<LeafletMap | null> = { current: null }
+
+    const { container } = render(
+      <Map vehicles={[vehicle]} reactLeafletRef={mapRef} stations={[station]} />
+    )
+
+    // Manual zoom
+    act(() => {
+      mapRef.current!.setZoom(15)
+    })
+    await animationFramePromise()
+    expect(container.innerHTML).toContain("m-station-icon")
```

Comment:
suggestion: would this work instead?
```suggestion
    expect(
      container.querySelector(".m-station-icon")
    ).toBeVisible()
```